Test-network.md

```diff
@@ -130,7 +132,7 @@ Step 2: Run storj sim with postgres
 ```
 // setup storj-sim network with postgres connection string. You can supply any password
 // and database name as long as they match the postgres instance running above
-$ storj-sim network --postgres=postgres://postgres@localhost/<dbName>?sslmode=disable setup
+$ storj-sim network setup --postgres=postgres://postgres@localhost/<dbName>?sslmode=disable
```

Comment:
Let's also add notices that `storj-sim network destroy` doesn't wipe the database. Also, maybe it should?